src/index/base.cpp

```diff
@@ -40,11 +40,6 @@ void BaseIndex::FatalErrorf(const char* fmt, const Args&... args)
     node::AbortNode(m_chain->context()->exit_status, message);
 }
 
-const CBlockIndex& BaseIndex::BlockIndex(const uint256& hash)
-{
-   return WITH_LOCK(cs_main, return *Assert(m_chainstate->m_blockman.LookupBlockIndex(hash)));
-}
-
```

Comment:
In f74b34f2:

`BaseIndex::BlockIndex` definition was removed but not its declaration in the header.